How to use this module with Node.js ?

Hello,

I try to use this module with node but i get the following error :

```shell
    Error: Cannot find module 'axis.js'
```

Step to reproduce :

```shell
    npm install axis.js
    node -e "console.log(require('axis.js'))"
```

any thoughts ?

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "author": "@toddmotto",
   "license": "MIT",
   "homepage": "https://github.com/toddmotto/axis",
+  "main": "dist/axis.js",
   "devDependencies": {
     "gulp": "~3.9.0",
     "gulp-uglify": "~0.3.0",

